ui_framework/components/tabs/tabs.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTab } from './tab';
+
+export const KuiTabs = ({
+  children,
+  selectedTabIndex,
+  onSelectedTabChanged,
+  className,
+  ...rest
+}) => {
+  const classes = classNames('kuiTabs', className);
+  const tabs = children.map((child,index)=>(
+    <KuiTab
+      key={index}
+      onClick={() => onSelectedTabChanged(index)}
+      isSelected={index === selectedTabIndex}
+    >
+      {child}
+    </KuiTab>
+  ));
+
+  return (
+    <div
+      className={classes}
+      {...rest}
+    >
+      {tabs}
+    </div>
+  );
+};
+
+const selectedTabIndexCheck = (props, propName, componentName) => {
+  const childrenLength = props.children ? props.children.length : 0;
+  const selectedTabIndex = props.selectedTabIndex;
```

Comment:
How about using the `propName` argument instead of hardcoding `selectedTabIndex`? Maybe we could even parameterize the use of `children` as in

```javascript
const propIndex = (indexedPropName) => (props, propName, componentName) => {
  const indexedPropLength = props[indexedPropName] ? props[indexedPropName].length : 0;
  const propValue = props[propName];

  // ...
};

KuiTabs.propTypes = {
  // ...
  selectedTabIndex: propIndex('children'),
  // ...
};
```

And maybe we want to include a type check for `Number` as well?